kubermatic/schema_cluster.go

```diff
@@ -156,14 +156,13 @@ func awsCloudSpecFields() map[string]*schema.Schema {
 		"access_key_id": {
 			Type:        schema.TypeString,
 			Required:    true,
-			DefaultFunc: schema.EnvDefaultFunc("AWS_ACCESS_KEY_ID", ""),
```

Comment:
should not we leave possibility to pass credentials over AWS envs? I think your upper change from `GetOkExists` to `GetOk` fixes the issue, right?

Reply:
as we spoke i will come with solution for passing the envs as it breaks thing in the current state